Makefile

```diff
@@ -14,3 +14,28 @@ $(MAKEFILE):
 	git clone --quiet --branch $(CI_VERSION) --depth 1 $(CI_REPOSITORY) $(CI_PATH);
 
 -include $(MAKEFILE)
+
+LD_FLAGS_INTEGRATION = -X main.version="integration-testing" -X main.build=$(BUILD) -X main.commit=$(COMMIT)
+
+GOTEST_INTEGRATION_TAGS_LIST = integration
+GOTEST_INTEGRATION_TAGS = $(GOTEST_INTEGRATION_TAGS_LIST)
```

Comment:
why not `GOTEST_INTEGRATION_TAGS = integration`?

Reply:
copied from lookout most probably. where we use `GOTEST_INTEGRATION_TAGS_LIST` to disable some tests on macos. Most probably not needed here

Reply:
> copied from lookout most probably

Exactly 😅. Gonna remove it.

Reply:
done